tap_asana/streams/teams.py

```diff
@@ -1,18 +1,33 @@
 
 from tap_asana.context import Context
-from tap_asana.streams.base import Stream, asana_error_handling
+from tap_asana.streams.base import Stream, asana_error_handling, REQUEST_TIMEOUT
 
 @asana_error_handling
 def find_team_by_organization(organization, opt_fields):
+  # Set request timeout to config param `request_timeout` value.
+  config_request_timeout = Context.config.get('request_timeout')
+  if config_request_timeout and float(config_request_timeout):
+    request_timeout = float(config_request_timeout)
+  else:
+    request_timeout = REQUEST_TIMEOUT # If value is 0,"0","" or not passed then it set default to 300 seconds.
+
   # Get and return a list of teams for provided organization
   teams = list(Context.asana.client.teams.find_by_organization(organization=organization,
-                                                               opt_fields=opt_fields))
+                                                               opt_fields=opt_fields,
+                                                               timeout=request_timeout))
   return teams
 
 @asana_error_handling
 def get_users_for_teams(team):
+  # Set request timeout to config param `request_timeout` value.
+  config_request_timeout = Context.config.get('request_timeout')
+  if config_request_timeout and float(config_request_timeout):
+    request_timeout = float(config_request_timeout)
+  else:
+    request_timeout = REQUEST_TIMEOUT # If value is 0,"0","" or not passed then it set default to 300 seconds.
```

Comment:
Put this comment above the if condition.

Reply:
Done